frontend/src/Student/components/ShowCard.js

```diff
@@ -160,15 +160,20 @@ const ShowCard = props => (
         {moment().isAfter(moment(props.show.entryEnd)) ? (
           <div>No Longer Accepting Submissions</div>
         ) : (
-            <div> Accepting Submissions Until:{' '}
-              <Moment format='MMMM Do YYYY'>{props.show.entryEnd}</Moment></div>
-          )}
+          <div>
+            {' '}
```

Comment:
I realize this has nothing to do with this PR, but this looks like an extraneous space.